qcore/im.py

```diff
@@ -153,16 +155,18 @@ def get_unit(self):
         elif self.name in [IMEnum.PGV, IMEnum.AI]:
             return "cm/s"
         elif self.name in [IMEnum.CAV, IMEnum.FAS]:
-            return "gs"  # FAS could also be cm/s depending on calculation / implementation
+            return (
+                "gs"  # FAS could also be cm/s depending on calculation / implementation
+            )
         elif self.name in [IMEnum.Ds575, IMEnum.Ds595, IMEnum.Ds2080]:
             return "s"
-        elif self.name in [IMEnum.MMI, IMEnum.IESDR]:
+        elif self.name in [IMEnum.MMI, IMEnum.SDI]:
```

Comment:
SDI is displacement, so should have units of m or cm?

Reply:
Well spotted. Thanks

Reply:
unit of cm, it is.